Description: fix for recent node-edia-type
 Since 1.0.0, node-media-type removed Content-Type parsing support. This
 patch updates code to be able to use node-type-is with it
Author: Xavier Guimard <yadd@debian.org>
Bug: https://github.com/jshttp/type-is/pull/43
Forwarded: https://github.com/jshttp/type-is/pull/43
Last-Update: 2019-11-21

--- a/index.js
+++ b/index.js
@@ -12,6 +12,7 @@
  * @private
  */
 
+var contentType = require('content-type')
 var typer = require('media-typer')
 var mime = require('mime-types')
 
@@ -236,13 +237,15 @@
 
 function normalizeType (value) {
   // parse the type
-  var type = typer.parse(value)
+  var type = contentType.parse(value)
 
   // remove the parameters
   type.parameters = undefined
 
   // reformat it
-  return typer.format(type)
+  var res = contentType.format(type)
+  if(!typer.test(res)){return false}
+  return res
 }
 
 /**
